Project.toml

```diff
@@ -30,4 +30,12 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Aqua", "Test", "Base64", "ReverseDiff", "SparseArrays", "StaticArrays", "Statistics"]
```

Comment:
Looks like `Aqua` expects this to be on a single line

Reply:
Now it should be fine